domain/build.gradle.kts

```
@@ -0,0 +1,9 @@
+plugins {
+    id("java-library")
+    id("org.jetbrains.kotlin.jvm")
```

```suggestion
    id("kotlin")
```
코틀린 모듈인 경우 다음으로도 충분합니다!